pom.xml

```diff
@@ -61,7 +62,7 @@
     <derby.version>10.10.2.0</derby.version>
     <beeju.version>1.2.1</beeju.version>
     <hamcrest-library.version>1.3</hamcrest-library.version>
-    <housekeeping.version>3.0.5</housekeeping.version>
+    <housekeeping.version>3.0.6-SNAPSHOT</housekeeping.version>
```

Comment:
Once housekeeping is released, I'll update this to 3.0.6, otherwise the version won't be resolved.